examples/shell/esp32/main/CMakeLists.txt

```diff
@@ -26,4 +26,4 @@ idf_component_register(SRCS main.cpp
                       PRIV_INCLUDE_DIRS
                       "${CMAKE_SOURCE_DIR}/third_party/connectedhomeip/src"
                       "${CHIP_SHELL_DIR}/shell_common/include"
-                      PRIV_REQUIRES chip nvs_flash bt console)
+                      PRIV_REQUIRES chip nvs_flash bt console esp32_mbedtls)
```

Comment:
What has changed that now this is required?

Reply:
the link will fail with shell-app after update idf version, so I add esp32_mbedtls component here.